Mathlib/RepresentationTheory/FDRep.lean

```diff
@@ -200,3 +201,41 @@ theorem dualTensorIsoLinHom_hom_hom : (dualTensorIsoLinHom ρV W).hom.hom = dual
 #align fdRep.dual_tensor_iso_lin_hom_hom_hom FDRep.dualTensorIsoLinHom_hom_hom
 
 end FDRep
+
+noncomputable section equivFullSubcategory
+
+namespace FDRep
+
+variable {k G : Type u} [Field k] [Monoid G] {V W : FDRep k G}
+
+/- TODO: Unify how to phrase lemmas/theorems - Use V? V.toRep? Or V.ρ.asModule? -/
+instance toRep_finiteDimensional : FiniteDimensional k V.toRep :=
+  FGModuleCat.instFiniteCarrier k _
+
+/-- Bundles Rep with a FiniteDimensional into a FDRep. -/
+def ofRep (V : Rep k G) [hV : FiniteDimensional k V] : FDRep k G :=
+  ⟨⟨V.V, hV⟩, V.ρ⟩
+
+/-- Functor from `FDRep` to the full subcategory of finite dimensional `Rep`. Also see
+`equivalenceFiniteDimensionalSubcategory` for an equivalence of category. -/
+def toFiniteDimensionalSubcategory :
+    FDRep k G ⥤ FullSubcategory (fun V : Rep k G ↦ FiniteDimensional k V) :=
+  FullSubcategory.lift _ (forget₂ _ _) inferInstance
+
+/-- Functor from the full subcategory of finite dimensional `Rep` to `FDRep`. Also see
+`equivalenceFiniteDimensionalSubcategory` for an equivalence of category. -/
+def ofFiniteDimensionalSubcategory :
+    FullSubcategory (fun V : Rep k G ↦ FiniteDimensional k V) ⥤ FDRep k G where
+  obj := fun ⟨V, _⟩ ↦ FDRep.ofRep V
+  map := fun f ↦ ⟨f.hom, f.comm⟩
+
+/-- Equivalence between `FDRep` and the full subcategory of finite dimensional `Rep`. -/
+def equivalenceFiniteDimensionalSubcategory :
```

Comment:
I think we usually make such things `equiv` not `equivalence`.

Reply:
You could probably also leave out Subcategory, naming this just `equivFiniteDimensionsal` (also above).